nautobot/extras/choices.py

```diff
@@ -96,6 +96,119 @@ class ButtonClassChoices(ChoiceSet):
     )
 
 
+#
```

Comment:
Prior to #3651, these choices were in `nautobot/core/choices.py` not `nautobot/extras/choices.py`; I think it makes more sense to restore them to that file as they're core Nautobot functionality, not an "extras" feature.